packages/snfoundry/package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "chain": "starknet-devnet --seed 0 --account-class cairo1",
     "deploy": "ts-node scripts-ts/helpers/deploy-wrapper.ts",
+    "deploy:verify": "cd contracts && sncast --url https://starknet-sepolia.public.blastapi.io/rpc/v0_7 verify --contract-address 0x3fdddc62c387905101e19d79e9a8b15adc015f5e1d6e2a8f2361fc3237e62b --contract-name YourContract --verifier walnut --network sepolia",
```

Comment:
this is a verification where an address is hardcoded and passed. 
The process has to have automatic verification of all contracts inside the `deploy.ts`, otherwise we cant achieve the target of this issue.